### DISCUSSION TITLE
[Crashlytics] Give ability to send bugs immediately using submitAllReports

### DISCUSSION THREAD

**Discussion by rodion-m** [firebase/flutterfire#6660] [Feature request] [answered]
Hello! Why not to give developers an ability to send an error immediately if they need it using internal `submitAllReports` method? \
This method: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java#L315

**Comment by rodion-m** [comment]
I think we should call someone from Firebase team here.
Hey @mrwillis21! Can you provide an api to call `submitAllReports`?

**Comment by robsonbbs** [comment]
This would be great!!
Oleg Kodysh said [here](https://stackoverflow.com/questions/62409353/firebase-sendunsentreports-sends-exceptions-only-after-restart) that this behavior is intended to save battery and bandwidth, I agree with that and that must be the default behavior, but I also agree that the developer must have the option to spend a little bit more battery and bandwidth if for some reason the report is needed right after being recorded.

**Comment by yyyy-ltd** [comment]
My problem is that some users have splash crash. and when user download app and getting splash crash he is going to delete the app. so i need to send it immediately and it cant wait

**Comment by mrober** [comment, marked as answer]
Crashlytics supports on-demand fatals for Flutter now, which will solve this case. PTAL at https://firebase.google.com/docs/crashlytics/customize-crash-reports?platform=flutter
